website/docs/r/vapp_vm.html.markdown

```diff
@@ -106,6 +106,65 @@ resource "vcd_vapp_vm" "web2" {
 
 ```
 
+
+## Example forced customization workflow
```

Comment:
What about moving this sample to the `## Customization` section below? This way it would appear contextually there where it's needed the most.

Reply:
True. I somehow forced myself into thinking that we put examples on top. Should be better now.